src/cpp/transport/TCPv4Transport.cpp

```diff
@@ -139,6 +139,40 @@ uint16_t TCPv4Transport::GetMaxLogicalPort() const
     return mConfiguration_.max_logical_port;
 }
 
+std::vector<std::string> TCPv4Transport::GetInterfacesList(const Locator_t& locator)
```

Comment:
Change to more intuitive name the function.

Reply:
Renamed as `GetBindingInterfacesList`